### DISCUSSION TITLE
It's possible to add select values on editOnClick() options?

### DISCUSSION THREAD

**Discussion by gallomaf** [Power-Components/livewire-powergrid#314] [Q&A] [answered]
Hello, every one.
I'm using this package and it is fantastic.

It is possible to add a select form with values on Coloumn with editOnClick()?

thank you in advance

**Comment by dansysanalyst** [comment, marked as answer]
ciao @gallomaf,

Thanks for the good words. It is not possible yet to use a select form on `editOnClick`.

The feature `selectOnClick` is being discussing and currently, it is in the backlog.

Would you consider contributing? PowerGrid is open for PRs!  🙂

**Reply by gallomaf** [reply]
I'm thinking on help on this feature, but I need some help to start. 
How I could start? Thank you

**Reply by dansysanalyst** [reply]
Hi @gallomaf,

Thank you so much for your interest! It's really nice to see the community engaging and enjoying the package. 

One of the main concerns is to have repeated `select inputs` loaded with many options on each table line. 

What else can we mention @luanfreitasdev? 😎

**Comment by gallomaf** [comment]
Thank You! I'm not sure I'm able to contribute.
However I will check the code and try
